docs/observability/how_to_guides/index.md

```diff
@@ -60,7 +60,6 @@ View and interact with your traces to debug your applications.
 Use LangSmith custom and built-in dashboards to gain insight into your production systems.
 
 - [Create and use custom dashboards](./how_to_guides/dashboards)
-- [Use built-in monitoring dashboards](./how_to_guides/use_monitoring_charts)
```

Comment:
- [Use built-in dashboards](./how_to_guides/dashboards#prebuilt-dashboards)
- [Create custom dashboards](./how_to_guides/dashboards#custom-dashboards)